src/components/views/messages/MessageActionBar.js

```diff
@@ -105,14 +112,22 @@ export default class MessageActionBar extends React.PureComponent {
 
     static contextType = RoomContext;
 
+    constructor(props) {
+        super(props);
+        this.elRef = React.createRef();
+        this.state = { focused: "options", buttonNames: [] };
+    }
+
     componentDidMount() {
         this.props.mxEvent.on("Event.decrypted", this.onDecrypted);
         this.props.mxEvent.on("Event.beforeRedaction", this.onBeforeRedaction);
+        this.elRef.current.addEventListener("keydown", this.onKeyDown, true);
```

Comment:
we should really be using react keydown handlers for this

(note to self mostly, as I am trying to make a generic Toolbar component now)